python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -211,17 +207,23 @@ def build_job(
 
         if not isinstance(selection, str):
             selection = check.opt_list_param(selection, "selection", of_type=str)
-        executor_def = check.opt_inst_param(executor_def, "executor_def", ExecutorDefinition)
+        executor_def = check.opt_inst_param(
+            executor_def, "executor_def", ExecutorDefinition, self.executor_def
+        )
         description = check.opt_str_param(description, "description")
+        resource_defs = {
```

Comment:
should we just to the "io_manager": fs_asset_io_manager thing here instead of in the group now that root manager is down here?

Reply:
Alas, the IO manager needs to be there for the `_validate_resource_reqs_for_asset_group` call right below.

Reply:
Not an insurmountable problem, but I kind of want to avoid getting to deep into it now.